### PR TITLE
feat: Add example of how to retrieve kubeconfig

### PR DESCRIPTION
**Pull request by mralexandernickel**

### CODE REVIEW

File: managed-k8s/README.md [hunk 1]
```diff
@@ -18,11 +18,38 @@ whereas each worker node has
 
 Provisioning time varies of course but we can expect to take around 10 minutes to spin up the cluster.
 
+## Retrieve kubeconfig
+
+Inside the cluster.tf you can see a defined output. the configured `value` needs
+to match the `gridscale_k8s` resource. In the given example, our `gridscale_k8s`
+resource is named `mycluster`:
+
+```
```
Comment (bkircher) [inline managed-k8s/README.md]: I think you can add `hcl` here for GH markdown to render it.

File: managed-k8s/README.md [hunk 2]
```diff
@@ -18,11 +18,38 @@ whereas each worker node has
...
+resource "gridscale_k8s" "mycluster" {
+  # ...
+}
+```
+
+So the configured value of our terraform `output` needs to look like this:
+
+```
```
Comment (bkircher) [inline managed-k8s/README.md]: Here too, `hcl`

File: managed-k8s/README.md [hunk 3]
```diff
@@ -18,11 +18,38 @@ whereas each worker node has
...
+output "kubeconfig" {
+  value = gridscale_k8s.mycluster.kubeconfig
+}
+```
+
+When you create your cluster terraform will write the kubeconfig now to STDOUT,
```
Comment (bkircher) [inline managed-k8s/README.md]: Can you make this lowercase "stdout" instead of STDOUT?

**Comment by bkircher** [comment]
Hey @mralexandernickel ! Thanks for the awesome PR. Can you try to make above minor changes. Will merge then.

**Comment by mralexandernickel** [comment]
Hi @bkircher ... Of course I would be willing to add some changes, but there is nothing to see, above your comment :) Could it be that you simply have a pending review? :)

**Comment by bkircher** [comment]
Haha, yes :smile:

**Review by bkircher — CHANGES_REQUESTED** [review]

File: managed-k8s/README.md (hunk 1 above)
Comment (mralexandernickel) [reply managed-k8s/README.md]: done

File: managed-k8s/README.md (hunk 2 above)
Comment (mralexandernickel) [reply managed-k8s/README.md]: done

File: managed-k8s/README.md (hunk 3 above)
Comment (mralexandernickel) [reply managed-k8s/README.md]: done

**Comment by mralexandernickel** [comment]
@bkircher I've added all the suggestions :)

**Comment by bkircher** [comment]
Awesome, thank you @mralexandernickel

**Pull request merged**